src/ajax.js

```diff
@@ -796,13 +796,14 @@ jQuery.each( [ "get", "post" ], function( i, method ) {
 			data = undefined;
 		}
 
-		return jQuery.ajax({
+		//extend to handle situation when url is actually a config object
```

Comment:
You could leave out this comment, it's pretty clear already. If it stayed you'd need to edit it to add a leading space and capitalize per the style guide.